crates/driver/src/infra/config/file/mod.rs

```diff
@@ -119,6 +122,29 @@ enum Mempool {
     },
 }
 
+pub mod encoding {
+    use {crate::domain::competition, serde::Deserialize};
+
+    /// Which logic to use to encode solutions into settlement transactions.
+    #[derive(Debug, Deserialize, Default)]
+    pub enum Strategy {
```

Comment:
nit: can we kebab-case this field? I think in the driver this is the most common casing.